test/integration/force_torque_plugin.cc

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <sdf/sdf.hh>
+
+#include <ignition/math/Helpers.hh>
+#include <ignition/sensors/ForceTorqueSensor.hh>
+#include <ignition/sensors/SensorFactory.hh>
+
+#include "test_config.h"  // NOLINT(build/include)
+#include "TransportTestTools.hh"
+
+/// \brief Helper function to create a force torque sdf element.
+sdf::ElementPtr ForceTorqueToSdf(const std::string &_name,
+    const ignition::math::Pose3d &_pose, const double _updateRate,
+    const std::string &_topic, const bool _alwaysOn,
+    const bool _visualize)
+{
+  std::ostringstream stream;
+  stream
+    << "<?xml version='1.0'?>"
+    << "<sdf version='1.6'>"
+    << " <model name='m1'>"
+    << "  <link name='link1'>"
+    << "    <sensor name='" << _name << "' type='force_torque'>"
+    << "      <pose>" << _pose << "</pose>"
+    << "      <topic>" << _topic << "</topic>"
+    << "      <update_rate>"<< _updateRate <<"</update_rate>"
+    << "      <alwaysOn>" << _alwaysOn <<"</alwaysOn>"
+    << "      <visualize>" << _visualize << "</visualize>"
+    << "    </sensor>"
+    << "  </link>"
+    << " </model>"
+    << "</sdf>";
+
+  sdf::SDFPtr sdfParsed(new sdf::SDF());
+  sdf::init(sdfParsed);
+  if (!sdf::readString(stream.str(), sdfParsed))
+    return sdf::ElementPtr();
+
+  return sdfParsed->Root()->GetElement("model")->GetElement("link")
+    ->GetElement("sensor");
+}
+
+// /// \brief Helper function to create an altimeter sdf element with noise
```

Comment:
remove ?

Reply:
Done in https://github.com/ignitionrobotics/ign-sensors/pull/144/commits/725290098fa4f8f4ffc17f805000806e02ab667a